### ISSUE TITLE
New module versions are sometimes selectable, but installing leads to an error

### ISSUE THREAD

**Issue by iron3oxide** [saltcorn/saltcorn#2735]
I had this issue with the fullcalendar module before, now it's the tabler module. When selecting the newest version to install (upgrade button was not available yet, but 0.5.1 was selectable so it's definitely my own fault), the installation fails saying that version cannot be found. That's not the end of the world, however when visiting the "?" menu of the module again, one is greeted with the following error message:

```
Error: Version 0.5.1 not found in available versions
    at doCheck (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/plugins-loader/stable_versioning.js:9:11)
    at isVersionSupported (/home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/plugins-loader/stable_versioning.js:66:10)
    at /home/saltcorn/.local/lib/node_modules/@saltcorn/cli/node_modules/@saltcorn/server/routes/plugins.js:1240:8
```

In case of the fullcalendar module, the upgrade button became available shortly after and everything worked fine again, so I'm hoping the same happens here. Not a huge priority IMHO, but ideally I'd expect only supported version to be selectable for installation.

**Comment by christianhugoch** [comment]
Thanks for spotting, it's a caching problem. As a workaround you could try to change the saltcorn installation version (up or down), start the server once, and then change the saltcorn version back to the desired value.

**Comment by christianhugoch** [comment]
should be fixed in [v1.0.0-beta.19](https://github.com/saltcorn/saltcorn/commit/55355a7e00aed3f7f1ef71a93e1d08aed6c4fbc8)

**Comment by iron3oxide** [comment]
I don't think I'll be able to confirm this soon as there would need to be a new version of a module I have installed. I'll reopen should I encounter it again. Thanks for the quick fix!
